src/stream/src/executor/backfill/arrangement_backfill.rs

```diff
@@ -34,7 +34,7 @@ use crate::executor::backfill::utils::{
     update_pos_by_vnode, BackfillProgressPerVnode, BackfillRateLimiter, BackfillState,
 };
 use crate::executor::prelude::*;
-use crate::task::CreateMviewProgress;
+use crate::task::CreateMviewProgressReporter;
```

Comment:
Rename this because the reported progress data structure is also called `CreateMviewProgress` and it's confusing